jpa/eclipselink.jpa.testapps.oracle/jpa.test.oracle.timestamptz/pom.xml

```diff
@@ -28,8 +28,8 @@
     <name>Test - oracle.timestamptz</name>
 
     <properties>
-        <ignore.se.failures>true</ignore.se.failures>
-        <ignore.server.failures>true</ignore.server.failures>
+        <ignore.se.failures>false</ignore.se.failures>
+        <ignore.server.failures>false</ignore.server.failures>
```

Comment:
false is default, so both should be just removed otherwise there is no way to control the behaviour from the parent pom

Reply:
OK, they were just removed.